app/src/main/java/swati4star/createpdf/util/ExcelToPDFAsync.java

```diff
@@ -37,8 +45,19 @@ protected void onPreExecute() {
     @Override
     protected Void doInBackground(Void... voids) {
         try {
-            final Workbook workbook = new Workbook(mPath);
-            workbook.save(mDestPath, FileFormatType.PDF);
+            LoadOptions loadOptions = new LoadOptions(FileFormatType.XLSX);
```

Comment:
Is the conversion (with password) working for `.xls` files ?

Reply:
Yes, it is working.
Line 48 is not of any use and I am removing it in my latest commit.